docs/dev/table/connectors/print.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Print SQL Connector"
+nav-title: Print
+nav-parent_id: sql-connectors
+nav-pos: 5
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<span class="label label-primary">Sink: Bounded</span>
+<span class="label label-primary">Sink: UnBounded</span>
+
+* This will be replaced by the TOC
+{:toc}
+
+The Print connector allows for writing every row to the standard output or standard error stream.
+
+It is designed for:
+
+- Easy test for streaming job.
+- Very useful in production debugging.
+
+Four possible format options:
+
+<table class="table table-bordered">
+    <thead>
+      <tr>
+        <th class="text-left" style="width: 40%">Print</th>
+        <th class="text-center" style="width: 30%">Condition1</th>
+        <th class="text-center" style="width: 30%">Condition2</th>
+      </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>PRINT_IDENTIFIER:taskId> output</h5></td>
+      <td>PRINT_IDENTIFIER provided</td>
+      <td>parallelism > 1</td>
+    </tr>
+    <tr>
+      <td><h5>PRINT_IDENTIFIER> output</h5></td>
+      <td>PRINT_IDENTIFIER provided</td>
+      <td>parallelism == 1</td>
+    </tr>
+    <tr>
+      <td><h5>taskId> output</h5></td>
+      <td>no PRINT_IDENTIFIER provided</td>
+      <td>parallelism > 1</td>
+    </tr>
+    <tr>
+      <td><h5>output</h5></td>
+      <td>no PRINT_IDENTIFIER provided</td>
+      <td>parallelism == 1</td>
+    </tr>
+    </tbody>
+</table>
+
+The output string format is "$row_kind(f0,f1,f2...)", row_kind is the short string of [RowKind](https://ci.apache.org/projects/flink/flink-docs-master/api/java/org/apache/flink/types/RowKind.html), example is: "+I(1,1)".
```

Comment:
The url should be changed with flink version. such as for 1.11, `flink-doces-master` should be changed to `flink-docs-release-1.11`

Reply:
Use { site.baseurl }

Reply:
Changed to `{{ site.baseurl }}`